graphql/course/mutations.ts

```diff
@@ -85,6 +85,21 @@ export class MutateCourseResolver {
         return result;
     }
 
+    @Mutation((returns) => GraphQLModel.Course)
+    @AuthorizedDeferred(Role.OWNER)
+    async courseDelete(@Ctx() context: GraphQLContext, @Arg('courseId') courseId: number): Promise<GraphQLModel.Course> {
+        const course = await getCourse(courseId);
+        await hasAccess(context, 'Course', course);
+        if (course) {
+            const result = await prisma.course.delete({ where: { id: courseId } });
```

Comment:
Move this to common/course

Also please implement checks for the prerequisites mentioned in https://github.com/corona-school/project-user/issues/1048